tools/modules.php

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env php
+<?php
+/**
+ * The modules.php script a database and automatically adds any new modules from
```

Comment:
It might be good to rename the script to be a verb like `update_modules_table` or whatever.

Reply:
I went with "manage_modules.php" because if there's ever more to do than update the table to install a module the script should take care of that too.